treasury/treasury/data.py

```diff
@@ -0,0 +1,141 @@
+import time
+import os
+import datetime
+
+import pandas as pd
+import requests
+from tqdm import tqdm
+
+XYM_API_HOST = os.getenv('XYM_API_HOST', 'wolf.importance.jp')
+XEM_API_HOST = os.getenv('XEM_API_HOST', 'alice5.nem.ninja')
+CM_KEY = os.getenv('CM_KEY' '')
+
+GECKO_TICKER_MAP = {
+   'ADA': 'cardano',
+   'AVAX': 'avalanche-2',
+   'BTC': 'bitcoin',
+   'DOT': 'polkadot',
+   'ETH': 'ethereum',
+   'LTC': 'litecoin',
+   'MATIC': 'matic-network',
+   'SOL': 'solana',
+   'TRX': 'tron',
+   'XEM': 'nem',
+   'XMR': 'monero',
+   'XYM': 'symbol',
+}
+
+XYM_MOSAIC_ID = '6BED913FA20223F8'
+MAX_TRIES = 6
+RETRY_S = 15
+
+
+def lookup_balance(address, asset):
+    asset = asset.lower()
+    if asset in ['symbol', 'xym']:
+        return lookup_xym_balance(address)
+    elif asset in ['nem', 'xem']:
+        return lookup_xem_balance(address)
+    else:
+        raise ValueError(f'Asset not supported: {asset}')
+
+
+def lookup_xym_balance(address):
+
+    balance = 0
+    json_account = requests.get('https://' + XYM_API_HOST + ':3001/accounts/' + address).json()
+    json_mosaics = json_account['account']['mosaics']
+    for json_mosaic in json_mosaics:
+        if XYM_MOSAIC_ID == json_mosaic['id']:
+            balance = float(json_mosaic['amount']) / 1000000
+        break
+
+    return balance
```

Comment:
With the current approach, we can track other assets other than XEM and XYM (via URLs for API clients in the config file). Given we don't want to just track XEM and XYM in the future, this would be ideal.

Alternatively, we could have a separate Client for each asset. That would require us to have familiarity with each asset.

What I would propose is instead to do a separate commit that uses both approaches - SymbolClient and NemClient for XYM and XEM, and API for other assets. When we want to grab things for more interactive purposes, we can add a client.